Samples/iOS15-SwiftUI/iOS15-SwiftUI.yml

```diff
@@ -0,0 +1,45 @@
+name: iOS15-SwiftUI
+createIntermediateGroups: true
+generateEmptyDirectories: true
+configs:
+  Debug: debug
+  Test: debug
+  TestCI: debug
+  Release: release
+projectReferences:
+  Sentry:
+    path: ../../Sentry.xcodeproj
+fileGroups:
+  - ../Shared/Config
+  - iOS15-SwiftUI.yml
+options:
+  bundleIdPrefix: io.sentry
+targets:
+  iOS15-SwiftUI:
+    type: application
+    platform: auto
+    sources:
+      - iOS15-SwiftUI
+      - ../Shared/GitInjections.swift
+      - ../Shared/screenshot.png
+      - ../Shared/SampleAssets.xcassets
+    dependencies:
```

Comment:
`m`: We could generate the `Info.plist` too

Reply:
https://github.com/getsentry/sentry-cocoa/pull/5166#discussion_r2074200222